genic/main.c

```diff
@@ -99,7 +112,7 @@ int main(int argc, char **argv)
       setup_grid(shift_nu, TotNu);
       displacement_fields(NuType);
       for(i = 0; i < NumPart; i++)
-          add_thermal_speeds(P[i].Vel);
+          add_thermal_speeds(&nu_therm, P[i].Vel);
```

Comment:
I recall Arka was talking about a scheme to drastically shrink the shotnoise level of these thermal speeds by coherently sampling the directions or something alike. He'll be around this Friday.

Reply:
You mean like this paper? https://arxiv.org/abs/1801.03906 That'd be a cool feature.

Reply:
Yes.